css/css-position/WEB_FEATURES.yml

```
@@ -0,0 +1,4 @@
+features:
+- name: absolute-positioning
+  files:
+  - '*absolute*'
```

The tests in this directory seem to often test the combination of different `position` values, and it's not straightforward to say what is the main feature under test. But I think this would be more conservative and less likely to match the wrong tests over time:

```suggestion
  - 'position-absolute-*'
```

Importantly, it's not going to match position-relative-*.

I've sent https://github.com/web-platform-tests/wpt/pull/55446 to fix a typo that would cause one test to be missed.